target/linux/ramips/dts/rt5350_zyxel_keenetic-lite_rev.b.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zyxel,keenetic-lite_rev.b", "ralink,rt5350-soc";
+	model = "ZyXEL Keenetic Lite rev.B";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status: power {
+			label = "zyxel:green:power";
```

Comment:
While the keenetic start is an exception, it's common to use the model part of the device name here, so "keenetic-lite-b:" instead of "zyxel:" if you follow my suggestion above.

Reply:
fixed